packages/flutter/test/widgets/sliver_list_test.dart

```diff
@@ -136,7 +136,7 @@ void main() {
       viewportHeight: viewportHeight,
     ));
     final int frames = await tester.pumpAndSettle();
-    expect(frames, greaterThan(1)); // ensure animation to bring tile17 into view
+    expect(frames, 1); // No animation when content shrinks suddenly.
```

Comment:
@goderbauer this is the breaking change here; what do you think, should we write a migration guide for it? It seems pretty minor to me and I'm not sure what I'd really put in a migration guide.

Reply:
(obviously if it turns out to break some customer tests that'll be a different question)

Reply:
If this doesn't break a customer tests I would be comfortable with not providing a migration guide.